oaknational/Oak-Web-Application
# Feat/lesq 276/downloads page copyright [LESQ-276]

## Description

For ticket [LESQ-276](https://www.notion.so/oaknationalacademy/Downloads-page-remove-header-card-and-add-conditional-copyright-text-display-e6ebeea4bcf04ce4bc3ed8c50b19e16c?pvs=4)  
  
- create `CopyrightNotice` component to render one of two copyright...copies, conditinally
- add storybook controls to enable toggling between the two
- add `CopyrightNotice` to `downloads` page
- update style of `downloads` page to match designs - removing header and divider lines & adjust spacing

note: a generated image file updated, I'm not sure why but I committed it in case it was needed 👀 

## How to test

1. Go to https://deploy-preview-1895--oak-web-application.netlify.thenational.academy/beta/teachers/programmes/design-technology-primary-ks1/units/structures-freestanding-structures-22b8/lessons/what-is-a-structure-6gw3jd/downloads?preselected=all 
2. You should see the pre-alb copyright message
3. go to https://deploy-preview-1895--oak-web-application.netlify.thenational.academy/beta/teachers-2023/programmes/geography-secondary-ks4/units/test-unit/lessons/lesson-1-in-test-unit/downloads
4. You should see the post-alb copyright message

## ACs
- [x]  Header card in the old style is nowhere to be seen
- [x]  Remove horizontal grey lines x 2
- [x]  If content is new, the “ALB” content message is shown
- [x]  If content is old, the “Pre-ALB” content message is shown
- [x]  Field needs to still be visible if school/email details are known and t’s and c’s have been previously accepted
